### DISCUSSION TITLE
password entry  show argument

### DISCUSSION THREAD

**Discussion by rakibsayed** [TomSchimansky/CustomTkinter#248] [Q&A]
is there a way to show and hide my password ,,,,, i couldn't be able to configure later to toggle hide and show password....i can only use the show argument when i created the widget

**Comment by Nomanpour** [comment]
To show the stars:
```
entry.configure(show='*')
```
to show the password:
```
entry.configure(show=None)
```

**Reply by rakibsayed** [reply]
thanks  ..... and does the cget()  method works with 'show' attributes in customtkinter.
